plugin/clang.vim

```diff
@@ -1287,7 +1287,7 @@ endf
 " Call clang-format to format source code
 func! s:ClangFormat()
   let l:view = winsaveview()
-  let l:command = printf("%s -style=%s ", g:clang_format_exec, g:clang_format_style)
+  let l:command = printf("%s -style=\"%s\" ", g:clang_format_exec, g:clang_format_style)
```

Comment:
There's already a default value for `g:clang_format_style` at [line 65](https://github.com/justmao945/vim-clang/blob/master/plugin/clang.vim#L65-L67) in the source code:

```
if !exists('g:clang_format_style')
  let g:clang_format_style = 'LLVM'
end
```